src/libraries/System.Net.Http/src/System/Net/Http/Headers/AltSvcHeaderParser.cs

```diff
@@ -0,0 +1,475 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Resources;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.Net.Http.Headers
```

Comment:
I thought AltSvc was going to be added separately?

Reply:
Change of plan; I was not happy with the design so rolled back and went with something simpler that only works with H3 to unblock. Will come back to it later